### PR TITLE
docs: web release notes

### PR DESCRIPTION
**Pull request by KaustubhKumar05**
Added content from https://github.com/100mslive/100ms-docs/pull/1825

### CODE REVIEW

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -14,6 +14,19 @@ description: Release Notes for 100ms JavaScript SDK
...
+### Added:
+- Roomkit Prebuilt: 
+    - Peers can now select virtual backgrounds to apply instead of getting a random background each time.
+    - Mweb header now indicates the type of audio device selected.
+
+- Roomkit Web: Added a new package that exports `HMSPrebuilt` from roomkit-react as a web component. [Read more](https://www.npmjs.com/package/@100mslive/roomkit-web)
+
+### Fixed:
+- Network indicator tooltip in the participant list would get hidden for the last peer.
+- Prebuilt: Preview would show go live CTA even when browser recording was running, resulting in failure to go live.
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:29]: check previous release notes. I think it is mentioned as Something else

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -14,6 +14,29 @@ description: Release Notes for 100ms JavaScript SDK
...
+## 2023-10-25
+Released: `@100mslive/hms-video-store@0.10.18`, `@100mslive/react-sdk@0.8.18`, `@100mslive/hls-player@0.1.18`, `@100mslive/roomkit-react@0.1.9`
+
+### Added:
+- Roomkit Prebuilt:  Polls and Quizzes
+- `hasNext` is now part of in [`usePaginatedParticipants`](/javascript/v2/react-hooks/interfaces/usePaginatedParticipantsResult)
+
+### Fixed:
+- Roomkit Prebuilt: Internal React Router is now removed. This will make it easy to use in next.js and remix
+- Roomkit Prebuilt: PDF share
```
[the remark this answers is not in the document]
Comment (raviteja83) [reply docs/javascript/v2/release-notes/release-notes.mdx:38]: @amar-1995 can we mention the specific change? should we just mention the fix for brave? what is the difference with updated code?

[the remark this answers is not in the document]
Comment (amar-1995) [reply docs/javascript/v2/release-notes/release-notes.mdx:38]: We need to say that prebuilt will now support PDF share.

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -14,6 +14,29 @@ description: Release Notes for 100ms JavaScript SDK
...
+Released: `@100mslive/hms-video-store@0.10.19`, `@100mslive/react-sdk@0.8.19`, `@100mslive/hls-player@0.1.19`, `@100mslive/roomkit-react@0.1.10`
+
+### Added:
+- Roomkit Prebuilt: 
+    - Peers can now select virtual backgrounds to apply instead of getting a random background each time.
+    - Mweb header now indicates the type of audio device selected.
+
+### Fixed:
+- Network indicator tooltip in the participant list would get hidden for the last peer.
+- Roomkit Prebuilt: Preview would show go live CTA even when browser recording was running, resulting in failure to go live.
```
Comment (amar-1995) [inline docs/javascript/v2/release-notes/release-notes.mdx:27]: We should add an introduction of states in hls, recording. We also had a functionality to pause/resume hls recording using API. @raviteja83 @KaustubhKumar05

Comment (raviteja83) [reply docs/javascript/v2/release-notes/release-notes.mdx:27]: that will be in the next week's release.

File: docs/javascript/v2/release-notes/release-notes.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (raviteja83) [reply docs/javascript/v2/release-notes/release-notes.mdx:38]: it was there before this release also btw. we just updated the internals.

**Review by raviteja83 — APPROVED** [review]

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**